test/powershell/engine/ExecutionPolicy.GroupPolicy.Tests.ps1

```diff
@@ -0,0 +1,39 @@
+﻿##
+## PowerShell ExecutionPolicy GroupPolicy tests
+## These are Windows only tests
+##
+
+Describe "User group policy execution policy should work" -Tags 'Feature' {
+
+    BeforeAll {
+
+        if (!$IsWindows) { return }
```

Comment:
Please use the Skip technique instead. https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk